moveit_core/robot_model/src/joint_model_group.cpp

```diff
@@ -734,6 +744,10 @@ void JointModelGroup::printGroupInfo(std::ostream& out) const
   out << "    ";
   for (int variable_index : variable_index_list_)
     out << variable_index << ' ';
+  out << "  * Active Variables Index List:\n";
+  out << "    ";
+  for (int active_variable_index : active_variable_index_list_)
+    out << active_variable_index << ' ';
```

Comment:
I think you need a newline `"\n"` at the end of this

Reply:
Good call. I moved it after the `(contiguous)` vs `(non-contiguous)` printout that was in the same line too.